Makefile

```
@@ -18,6 +18,8 @@ CLEAN_IMG ?= ${IMAGE_PREFIX}cleanup:${TEST_INFRA_VERSION}
 #${IMAGE_PREFIX}cleanup_agent:${TEST_INFRA_VERSION}
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
+# Set timeout for ready container
+READY_TIMEOUT_ARG ?=20m
```

nit: Add a space between equal and quote to escape any system that may interpret it as an integer.

```suggestion
READY_TIMEOUT_ARG ?= "20m"
```

---

Thank you for pointing that out!